app/views/buddies/show.html.erb

```diff
@@ -30,8 +30,17 @@
       <% if current_user.id == review.user_id %>
         <%= link_to "Edit Review", edit_buddy_review_path(@buddy, review) %>
         <%= link_to "Delete Review", buddy_review_path(@buddy, review), method: :delete %>
+      <% end %>
 
+      <%= form_for([@buddy, review, review.votes.build]) do |f| %>
```

Comment:
You don't need to nest the resources this deep. In your controller you only need the review and the vote:

```HTML+ERB
<%= form_for [@buddy, review.votes.build] do |f| %>
```